modules/apps/commerce/commerce-product-content-web/src/main/resources/com/liferay/fragment/renderer/dynamic_field/dependencies/configuration.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
Move this file under com/liferay/commerce/product/content/web/internal/fragment/renderer/dynamic_field/dependencies